views/layouts/main.handlebars

```diff
@@ -5,11 +5,11 @@
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title> {{ title }} </title>
+	<script defer src='../javascripts/search-bar.js'></script>
```

Comment:
One more thing now that I see this, lets use an absolute path instead of a relative one. This might work on `cmdmethods.com/home`, but it will break when using it anywhere else, for example `cmdmethods.com/cards/some-method`